tests/source/issue-4427.rs

```diff
@@ -0,0 +1,10 @@
+const A: usize = 
+  // Some constant
+    2;
+
+const B: usize =   
+/* constant */
+3;
+
+const C :   usize   
+      = /* foo */5;
```

Comment:
Let's add a few more test scenarios just to cover some additional bases. It doesn't necessarily need to be these exact snippets (though certainly can be!) just as long as the same scenarios are included.

```suggestion
      = /* foo */5;
      
      const D: usize =    // baz
/*    Some constant */
            /* ba */
 {  3  
 // foo   
    };

const E:   usize=             /* foo */5;

const F: usize =
{
        7
    };

const G: usize = /* foooooooooooooooooooooooooooooooooooooooooooooooooooooooo0000000000000000xx00 */ 5;
                  const H: usize = /* asdfasdf */ match G > 1 {
                    true => 1,
                    false => 3,
                };
            
                pub static FOO_BAR: Vec<u8> =    //f
                {
                    vec![]};
```